src/uvw/emitter.hpp

```diff
@@ -178,7 +177,9 @@ class Emitter {
      */
     void clearAll() noexcept {
         for(auto &&h: handlers) {
-            h->clear();
+            if (h) {
```

Comment:
This is a bug fix.
You should rather create two PR when you want to fix something and to propose changes.
This way we can speed up one PR and review the other one.